debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/event_handler/rdbms/RdbmsDeleteEvent.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink.event_handler.rdbms;
```

Comment:
A nitpick, but I'd prefer we use the term `relational` rather than `rdbms`, just to remain consistent with other classes and packages that are associated with relational databases.  

This should also apply to class naming, not just the package name.

Reply:
It's not nitpick but a great syntactical comment!
done